app/services/submit_application.rb

```diff
@@ -79,6 +79,8 @@ def enough_references_have_been_provided?
   def edit_by_time
     if HostingEnvironment.sandbox_mode?
       Time.zone.now
+    elsif application_form.candidate_has_previously_applied?
```

Comment:
I'm going to look at this in the refactor, I suspect this isn't actuallu necessary. But otherwise the tests fail.